app/transforms/file.js

```diff
@@ -1,6 +1,5 @@
 import Ember from 'ember';
 import DS from 'ember-data';
-import getOwner from 'ember-getowner-polyfill';
```

Comment:
This is the only occurrence, so the polyfill can also be removed from the package in that case

Reply:
So you want this line to remain?